data/abilities.js

```diff
@@ -282,7 +282,10 @@ exports.BattleAbilities = {
 		desc: "If this Pokemon eats a Berry, it restores 1/3 of its maximum HP, rounded down, in addition to the Berry's effect.",
 		shortDesc: "If this Pokemon eats a Berry, it restores 1/3 of its max HP after the Berry's effect.",
 		onEatItem: function (item, pokemon) {
-			this.heal(pokemon.maxhp / 3);
+			let target = pokemon.side.foe.active[pokemon.side.foe.active.length - 1 - pokemon.position];
```

Comment:
All opposing Pokemon with unnerve block berry consumption and you can't just check one of the Pokemon.